a3p-integration/proposals/a:upgrade-next/yarn.lock

```diff
@@ -335,6 +335,16 @@ __metadata:
   languageName: node
   linkType: hard
 
+"brace-expansion@npm:^1.1.7":
```

Comment:
all these changes to `yarn.lock` come from adding `tmp`?

I guess this is not our main `yarn.lock`, so there's not much supply-chain risks.

Reply:
removed